net-im/signal-desktop-bin/signal-desktop-bin-1.30.0-r1.ebuild

```diff
@@ -40,6 +41,8 @@ src_prepare(){
 	sed -e 's|\("/opt/Signal/signal-desktop"\)|\1 --start-in-tray|g' \
 		-i usr/share/applications/signal-desktop.desktop || die
 	unpack usr/share/doc/signal-desktop/changelog.gz
+	# Fix Bug 706352
+	chrpath opt/Signal/resources/app.asar.unpacked/node_modules/sharp/vendor/lib/libjpeg.so.8.2.2 -r '$ORIGIN:/target/lib'
```

Comment:
Every external command need to be appended with `|| die`, like with the sed call above.

Reply:
done